sympy/stats/rv.py

```diff
@@ -757,7 +757,11 @@ def expectation(expr, condition=None, numsamples=None, evaluate=True, **kwargs):
     from sympy.stats.symbolic_probability import Expectation
     if evaluate:
         return Expectation(expr, condition).doit(**kwargs)
-    return Expectation(expr, condition).rewrite(Integral) # will return Sum in case of discrete RV
+    message = ("Using `evaluate=False` now returns `Expectation` object "
+              "since version 1.7. If you want unevaluated Integral/Sum use "
+              "`E(expr, condition, evaluate=False).rewrite(Integral)`")
```

Comment:
```suggestion
    message = ("Since version 1.7, using `evaluate=False` returns `Expectation` object. "
              "If you want unevaluated Integral/Sum use "
              "`E(expr, condition, evaluate=False).rewrite(Integral)`")
```